src/main/java/com/bieganski/ox/App.java

```diff
@@ -2,6 +2,18 @@
 
 public class App{
 	public static void main(String[] args){
-		System.out.println("Hello world!");
+		UserInterface userInterface = new ConsoleInterface(System.in, System.out);
+		userInterface.print("Tic tac toe!");
+		Board board = new Board(15);
+		try {
+			board.addFiled(new Field(new Coordinates(2, 2), Symbol.X));
+			board.addFiled(new Field(new Coordinates(6, 7), Symbol.O));
+			board.addFiled(new Field(new Coordinates(2, 1), Symbol.X));
+			board.addFiled(new Field(new Coordinates(3, 5), Symbol.O));
```

Comment:
Field or Filled was meant here? That is a bit peculiar, that you are adding Fields to the Board. Does it need to grow in some way DURING the game? Or is it filling the field perhaps or occupying it?